tutorial.md

```diff
@@ -22,6 +22,18 @@ It also can test the output to make sure everything ran as planned.
 Hello world
 ```
 
+# Test Code block contains expected output
+
+```azurecli-interactive
+echo "Foo Bar"
+```
+
+It also can test the output to make sure everything ran as planned.
+<!--expected_contains="Foo"-->
```

Comment:
I suggest `expected_substring` instead of `expected_contains` as it sounds odd to me, doesn't it for you?

Reply:
We do not want to add additional tags unnecessarily, this is a barrier to adoption as users will need to learn the different approaches. I can't think of any reason why we can't infer the kind of similarity test from the string provided. Therefore, we should use the existing `expected_similarity`.

That said, this is an opportunity to review the one tag to rule them all. We are early enough that I'd be OK with deprecating `expected_similarity` for something more inclusive. I'm only against introducing a second one.

Reply:
Following up on this comment, if we look at #14 we see a request for Regex matching. For this proposal we can implement "contains" through a regex and/or wildcard (e.g. `*foo*`).